### DISCUSSION TITLE
Switch Your React App Between Material, Bootstrap and Custom Themes at Runtime

### DISCUSSION THREAD

**Discussion by Shivansh-yadav13** [EddieHubCommunity/support#2837] [Ideas]
- Idea is to switch between custom Primereact themes
- Idea suggested by @schmelto 
- resource https://primetek.hashnode.dev/switch-your-react-app-between-material-bootstrap-and-custom-themes-at-runtime
- I have completed all the steps, but after doing `npm start` I am facing this issue

![Screenshot 2021-11-03 192614](https://user-images.githubusercontent.com/87603425/140073475-15df7472-c188-4866-9ec0-adc001ac8224.png)

**Comment by eddiejaoude** [comment]
Just seeing this. Interesting idea, did you solve it?

**Reply by Shivansh-yadav13** [reply]
I'll look for the solution again and will let you know soon :)
